### PR TITLE
[sonyprojector] Add discovery through SDDP

### PR DESCRIPTION
**Pull request by lolodomo**
Only applicable to projector models having a network connector

Not tested because my projector has no network connector

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by mlobstein** [comment]
The configuration option to enable SDDP in the projector is called 'Start SDDP Service'
https://helpguide.sony.net/vpl/xw5000/v1/en/contents/TP1000558242.html


Suggested verbiage for the **Discovery** section in the README:

_If the projector is connected via Ethernet and the 'Start SDDP Service' option is present and enabled in the projector's Advanced Settings->Service page, the thing using PJ Talk will be discovered automatically.
Serial or Serial over IP connections must be configured manually._

**Review by mlobstein — CHANGES_REQUESTED** [review]
LGTM except needing the README update

**Comment by lolodomo** [comment]
> LGTM except needing the README update

Done now.

**Review by mlobstein — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.sonyprojector/README.md [hunk 1]
```diff
@@ -94,8 +94,8 @@ This binding supports the following thing types:
 
 ## Discovery
 
-Discovery is not supported at the moment.
-You have to add all things manually.
+If the projector is connected via Ethernet and the 'Start SDDP Service' option is present and enabled in the projector Advanced Settings->Service page, the thing using Ethernet connection and PJ Talk will be discovered automatically.
```
Comment (jlaur) [inline bundles/org.openhab.binding.sonyprojector/README.md:97]:
```suggestion
If the projector is connected via Ethernet and the 'Start SDDP Service' option is present and enabled in the projector Advanced Settings->Service page, the Thing using Ethernet connection and PJ Talk will be discovered automatically.
```

Comment (lolodomo) [reply bundles/org.openhab.binding.sonyprojector/README.md]: Changed

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**